camkes/templates/component.common.c

```diff
@@ -84,6 +84,31 @@ int get_instance_affinity(void) {
     return /*? configuration[me.name].get('affinity', options.default_affinity) ?*/;
 }
 
+#ifdef CONFIG_ALLOW_SMC_CALLS
+seL4_CPtr camkes_get_smc_cap(seL4_Word smc_call){
+    seL4_CPtr cap;
+    switch(smc_call) {
+/*# ARM SMC cap allocation #*/
+/*- if 'allow_smc' in configuration[me.name].keys() -*/
+    /*- if configuration[me.name].get('allow_smc') == true -*/
```

Comment:
* merge the lines and use a default value `if configuration[me.name].get('allow_smc', false)`, basically the same concept as used in line 84. And is saves a level of indention, that improves readability a bit.
* could drop the explicit check `== true` then also.